tox.ini

```diff
@@ -0,0 +1,57 @@
+[tox]
+minversion = 4.0.0
+envlist = linters
+ignore_base_python_conflict = True
+
+[testenv]
+basepython = python3
+usedevelop = True
+passenv = *
+setenv =
+   ANSIBLE_SKIP_CONFLICT_CHECK=1
+   ANSIBLE_ACTION_PLUGINS={toxinidir}/edpm_ansible/roles.galaxy/config_template/action:{toxinidir}/edpm_ansible/ansible_plugins/action
+   ANSIBLE_CALLBACK_PLUGINS={toxinidir}/edpm_ansible/ansible_plugins/callback
+   ANSIBLE_FILTER_PLUGINS={toxinidir}/edpm_ansible/ansible_plugins/filter
+   ANSIBLE_LIBRARY={toxinidir}/edpm_ansible/roles.galaxy/config_template/library:{toxinidir}/edpm_ansible/ansible_plugins/modules
+   ANSIBLE_MODULE_UTILS={toxinidir}/edpm_ansible/ansible_plugins/module_utils
+   ANSIBLE_ROLES_PATH={toxinidir}/edpm_ansible/roles.galaxy:{toxinidir}/edpm_ansible/roles
+   ANSIBLE_INVENTORY={toxinidir}/tests/hosts.ini
+   ANSIBLE_NOCOWS=1
+   ANSIBLE_RETRY_FILES_ENABLED=0
+   ANSIBLE_STDOUT_CALLBACK=debug
+   ANSIBLE_LOG_PATH={envlogdir}/ansible-execution.log
+   VIRTUAL_ENV={envdir}
+   LC_ALL=en_US.UTF-8
+   # pip: Avoid 2020-01-01 warnings: https://github.com/pypa/pip/issues/6207
+   # paramiko CryptographyDeprecationWarning: https://github.com/ansible/ansible/issues/52598
+   PYTHONWARNINGS=ignore:DEPRECATION::pip._internal.cli.base_command,ignore::UserWarning
+   PIP_DISABLE_PIP_VERSION_CHECK=1
+   TRIPLEO_ANSIBLE_COMPUTE_NODE_MOLECULE_CACHE={homedir}/.cache/edpm-ansible/containers
+   TRIPLEO_ANSIBLE_COMPUTE_NODE_MOLECULE_VOLUMES=['{homedir}/.cache/edpm-ansible/containers:/var/lib/containers:rw','/sys/fs/cgroup:/sys/fs/cgroup:rw']
+sitepackages = True
+deps =
+   -r {toxinidir}/requirements.txt
+   -r {toxinidir}/test-requirements.txt
+   -c {env:TOX_CONSTRAINTS_FILE:https://releases.openstack.org/constraints/upper/master}
```

Comment:
we may want to use zed u-c for the time being ?

Reply:
ack.